cmd/crc/cmd/start.go

```diff
@@ -75,32 +75,38 @@ func runStart(ctx context.Context) (*types.StartResult, error) {
 		KubeAdminPassword: config.Get(crcConfig.KubeAdminPassword).AsString(),
 		Preset:            crcConfig.GetPreset(config),
 		EnableSharedDirs:  crcConfig.ShouldEnableSharedDirs(config),
+		IngressHTTPPort:   config.Get(crcConfig.IngressHTTPPort).AsUInt(),
+		IngressHTTPSPort:  config.Get(crcConfig.IngressHTTPSPort).AsUInt(),
 	}
 
 	client := newMachine()
 	isRunning, _ := client.IsRunning()
 
 	if !isRunning {
 		if err := checkDaemonStarted(); err != nil {
-			return nil, err
+			return nil, "", err
 		}
 
 		if err := preflight.StartPreflightChecks(config); err != nil {
-			return nil, crcos.CodeExitError{
+			return nil, "", crcos.CodeExitError{
 				Err:  err,
 				Code: preflightFailedExitCode,
 			}
 		}
 	}
+	portFallBackWarning := portFallbackWarning(config.Get(crcConfig.IngressHTTPPort).AsUInt(),
+		config.Get(crcConfig.IngressHTTPSPort).AsUInt())
 
-	return client.Start(ctx, startConfig)
+	result, err := client.Start(ctx, startConfig)
+	return result, portFallBackWarning, err
```

Comment:
nit: `portFallBackWarning` can directly be called inside `renderStartResult` the `runStart` function doesn't necessarily have to generate and return it

Reply:
@anjannath we don't have config option in `renderStartResult` which is used to get what http/https port user specified before generating the warning.

Reply:
you mean `config`? it should be accessible to the whole `cmd` package

Reply:
Ah yes, then it should be much simpler, Thanks 👍🏼